src/components/MentorBecomeCard.tsx

```diff
@@ -0,0 +1,77 @@
+import { Grid, Typography, useMediaQuery, Button } from '@mui/material';
```

Comment:
The overall title is too specific - might be confusing - call it something simpler please

Reply:
I changed the name of the prop to `mentorOrMentee`. Does that work? :)